forms-flow-api/src/formsflow_api/models/form_logs.py

```diff
@@ -0,0 +1,49 @@
+"""This is form logs model."""
+
+from sqlalchemy.dialects.postgresql import ARRAY, JSON
+from .base_model import BaseModel
+from .db import db
+
+
+class FormLogs(BaseModel, db.Model):
+    """This is for creating form logs."""
+
+    id = db.Column(db.Integer, primary_key=True)
+    form_id = db.Column(db.String(30), nullable=False)
+    logs = db.Column(ARRAY(JSON), nullable=False)
+
+    @classmethod
+    def create_form_log(cls, data):
+        """This method will create form logs in db."""
+        form_log = FormLogs()
+        form_log.form_id = data["form_id"]
+        form_log.logs = data["logs"]
+        form_log.save()
+        return form_log
+
+    @classmethod
+    def get_form_logs(cls, form_id):
+        """Return the logs against form id."""
+        if form_id:
+            form_logs = cls.query.filter(cls.form_id == form_id).first()
+            if form_logs:
+                return form_logs
+        return None
+
+    @classmethod
+    def update_form_logs(cls, form_id, data):
+        """Update form logs against form id."""
+        if form_id:
+            form_logs = cls.query.filter(cls.form_id == form_id).first()
```

Comment:
assert form_logs too, as it doesn't make sense to return None for an operation which is supposed to fail.